### PR TITLE
login as customer set as coding standard

### PR DESCRIPTION
**Pull request by priyankavasoya**
@

### CODE REVIEW

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (harishmk1805) [inline app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml:1]: remove unused space

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */
+?>
+<?php
+/**
+ * @var $block \Magento\Framework\View\Element\Template
+ */
+?>
+<p>
+    <?=$escaper->escapeHtml(__('You have logged in and will go to customer account in 5 seconds.')); ?>
```
Comment (harishmk1805) [inline app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml:23]: <?=  $escaper->escapeHtml(

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+</p>
+<form id="loginascustomer" method="post" action="<?=$escaper->escapeHtml(__($block->getUrl('*/*/post'))); ?>">
```
Comment (harishmk1805) [inline app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml:25]: <?= $escaper->escapeHtml(

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <button style="display: none;" id="loginascustomer-button" type="submit" class="action save primary" title="">
+        <span> >> </span>
```
Comment (harishmk1805) [inline app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml:27]: >>   ??

File: app/code/Dss/LoginAsCustomer/view/frontend/web/js/login.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (harishmk1805) [inline app/code/Dss/LoginAsCustomer/view/frontend/web/js/login.js:1]: remove unused space

**Review by harishmk1805 — CHANGES_REQUESTED** [review]

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml (hunk 1 above)
Comment (priyankavasoya) [reply app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml]: Done

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml (hunk 2 above)
Comment (priyankavasoya) [reply app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml]: done

File: app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml (hunk 4 above)
Comment (priyankavasoya) [reply app/code/Dss/LoginAsCustomer/view/frontend/templates/login.phtml:26]: done

File: app/code/Dss/LoginAsCustomer/view/frontend/web/js/login.js (hunk 5 above)
Comment (priyankavasoya) [reply app/code/Dss/LoginAsCustomer/view/frontend/web/js/login.js]: done

**Review by harishmk1805 — APPROVED** [review]

**Pull request merged**